### ISSUE TITLE
Resource Management - Add new resource function

### ISSUE THREAD

**Issue by Ardianpalloshi** [amosproj/amos2022ws05-shared-desk-mgmt#18]
## User story
1. As a user (admin)
2. I want / need to be able to add new resources available to be booked
3. So that the employees of my company can book them

## Acceptance criteria
* The function should be accessible through the resource management's "add resource" button and be initialized after completing entering all of the resource information and hitting the respective button
* The function should add an appropriate entry in the database with the new resource information
* After adding the resource it should be displayed in the list of all available resources of that company in the admin section
* After adding the resource it should be bookable by users of the respective company

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
